ansible/inventory/group_vars/defaults.yaml

```diff
@@ -17,6 +17,7 @@ aws:
   aem_artifacts_base: overwrite-me
   aem_certs_base: overwrite-me
   install_ssm_agent: true
+  install_cloudwatchlogs: true
```

Comment:
Please update configuration document with this property. https://github.com/shinesolutions/packer-aem/blob/master/docs/configuration.md

Reply:
Done